runtime/onert/backend/train/ops/ConvolutionLayer.cc

```diff
@@ -134,21 +134,16 @@ void ConvolutionLayer::backwardFloat32()
 {
```

Comment:
### for reviewers

This PR is much easier to review with split diff view.

Reply:
Is there a difference from https://github.com/Samsung/ONE/pull/12492/files?

Reply:
> Is there a difference from [#12492 (files)](https://github.com/Samsung/ONE/pull/12492/files)?

There is no difference! 
It just forces to have a split view. (not unified view) 
( It looks to cause some confusion. So I removed the line )